lib/model/model.go

```diff
@@ -2426,6 +2429,7 @@ func (m *model) CommitConfiguration(from, to config.Configuration) bool {
 				l.Infoln("Adding folder", cfg.Description())
 				m.newFolder(cfg)
 			}
+			m.closeConns(cfg.DeviceIDs(), errFolderShared)
```

Comment:
newFolder calls closeConns, and a new paused folder shouldn't require closing

Reply:
It would in theory. For consistency we should send cc with paused=true